Sources/DocCDocumentation/DocCDocumentation.docc/Purchases.md

```diff
@@ -115,13 +115,14 @@ Most features require configuring the SDK before using it.
 - ``Purchases/setOnesignalID(_:)``
 
 ### Advanced Configuration
-- ``Purchases/finishTransactions``
```

Comment:
Was this removal intentional? I guess we now need to set this up with observer mode, so this is probably ok. Just confirming we don't want to keep the old one.

Reply:
Oh yeah forgot to mention this. But in going to explain it I realized my logic was flawed, so let me revert it.